addons/website_forum/models/forum.py

```diff
@@ -139,6 +140,7 @@ def _compute_last_post(self):
         for forum in self:
             forum.last_post_id = forum.post_ids.search([('forum_id', '=', forum.id), ('parent_id', '=', False), ('state', '=', 'active')], order='create_date desc', limit=1)
 
+    @api.depends_context('lang')
```

Comment:
Don't add depends_context on the compute of a stored field, at best it won't work, at worst you'll trigger a "write" call every time someone opens the page.

It should be a separate compute, if anything.